WooCommerce/WooCommerceTests/ViewRelated/Themes/ThemesPreviewViewModelTests.swift

```diff
@@ -0,0 +1,65 @@
+import XCTest
+@testable import WooCommerce
+@testable import Yosemite
+
+final class ThemesPreviewViewModelTests: XCTestCase {
+    private var stores: MockStoresManager!
+
+    override func setUp() {
+        super.setUp()
+        stores = MockStoresManager(sessionManager: SessionManager.makeForTesting())
+    }
+
+    override func tearDown() {
+        stores = nil
+        super.tearDown()
+    }
+
+
+    func test_initial_state_is_pagesLoading() {
+        // Given
+        let viewModel = ThemesPreviewViewModel(themeDemoURL: "https://tsubakidemo.wpcomstaging.com/")
+
+        // Then
+        XCTAssertEqual(viewModel.state, .pagesLoading)
+        XCTAssertEqual(viewModel.pages.count, 1) // The default "Home" page.
```

Comment:
1. You can test the actual contents of the `pages` array to ensure the value is "Home" page. (Instead of checking  the `count`)
1. I recommend moving this "Home" page check to a separate unit test. It is better if one test checks for one thing. It helps to identify issues when a test fails.

Reply:
I kept the `count` checking because I think it's useful as a check as well.